### PR TITLE
fix: remove filesystemperformance from default.yaml

### PR DESCRIPTION
**Pull request by xavpaice**
Removes filesystemPerformance from the default host collector spec.

### CODE REVIEW

File: host/default.yaml [hunk 1]
```diff
@@ -352,20 +352,6 @@ spec:
         collectorName: "ps-high-load"
         command: "sh"
         args: ["-c", "ps -eo s,user,cmd | grep ^[RD] | sort | uniq -c | sort -nbr | head -20"]
-    - filesystemPerformance:
-        collectorName: filesystem-latency-two-minute-benchmark
-        timeout: 2m
```
Comment (CpuID) [inline host/default.yaml:357]: do we keep it but just reduce the timeout way down? 20-30sec instead of 2min?

**Review by CpuID — COMMENTED** [review]
just one thought as part of my review... :)

File: host/default.yaml (hunk 1 above)
Comment (chris-sanders) [reply host/default.yaml:357]: I like that, I have no idea why we would _need_ 2 minutes.

**Comment by xavpaice** [comment]
The spec actually excludes this collector/analyzer, and just allows folks to see it as a template to be included if they want.  I think, given that, we can just close this PR and not worry about it.

**Pull request closed without merging**